backend/dao/winesDAO.js

```diff
@@ -190,21 +184,19 @@ module.exports = class WinesDAO {
       console.error(`Unable to issue find query: ${e}`);
       return { singleWine: 0 };
     }
-    const displayCursor = cursor.limit(winesPerPage).skip(winesPerPage * page);
+    //const displayCursor = cursor.limit(winesPerPage).skip(winesPerPage * page);
 
     try {
-      const winesList = await displayCursor.toArray();
-      console.log(winesList);
-      const totalWines = await wines.countDocuments(query);
-      console.log("total wines:", totalWines);
+      const singleWine = await cursor.toArray();
+
       //*tblshooting, empty the query here..
       //todo: test more usecases for problems
       query = {};
 
-      return { winesList, totalWines };
+      return { singleWine };
     } catch (e) {
       console.error(`Unable to convert cursor to array: ${e}`);
-      return { wineList: [], totalWines: 0 };
+      return { singleWine: 0 };

```

Comment:
In this and the other case above we should return `null`.

Reply:
will fix!